accessories/factory.js

```diff
@@ -427,6 +433,30 @@ module.exports = homebridge => {
     get numberOfPowerMeters() {
       return 0
     }
+
+    get numberOfAccessories() {
+      if (this.config.type === 'thermostat') {
```

Comment:
Unfortunately I can't find out without the settings how many sensors are connected at only the device state. Maybe it's also useful for other accessories to have the config inside the factory. Is there a better way?